### PR TITLE
Update. JS. Handle external forms in frames.

### PR DESCRIPTION
**Pull request by svfcode**

### CODE REVIEW

**Review by ArtemAnoshin — APPROVED** [review]

**Review by Glomberg — APPROVED** [review]

File: js/src/apbct-public-bundle.js [hunk 1]
```diff
@@ -2681,6 +2680,11 @@ function apbctProcessIframes()
 
 function apbctProcessExternalForm(currentForm, iterator, documentObject) {
 
+    //skip excluded forms
+    if ( formIsExclusion(currentForm)) {
```
Comment (alexandergull) [inline js/src/apbct-public-bundle.js:2684]: Already checked there https://github.com/CleanTalk/wordpress-antispam/blob/dbe309ce43b0867e5124ebeb47b2a09eae591470/js/src/apbct-public--5--external-forms.js#L130
Isnt it?

File: js/src/apbct-public-bundle.js [hunk 2]
```diff
@@ -2983,6 +2987,7 @@ function sendAjaxCheckingDinamicFormData(form) {
     var visible_fields = {};
     visible_fields[0] = apbct_collect_visible_fields(form);
     apbct_visible_fields_set_cookie( visible_fields );
+    form.append(ctNoCookieConstructHiddenField('hidden'));
```
Comment (alexandergull) [inline js/src/apbct-public-bundle.js:2990]: This way is for NoCookie way. What about if cookies is native or alternative?

File: js/src/apbct-public-bundle.js (hunk 1 above)
Comment (svfcode) [reply js/src/apbct-public-bundle.js:2684]: You compare bundle version with part of it.

File: js/src/apbct-public-bundle.js (hunk 2 above)
Comment (svfcode) [reply js/src/apbct-public-bundle.js:2990]: This append resolve NoCookie way in iframe cases. For native or alternative way we could make research.

**Pull request merged**